internal/controller/scheduler/scheduler.go

```diff
@@ -90,9 +96,12 @@ func (scheduler *Scheduler) Run() {
 
 func (scheduler *Scheduler) reportStats(workers []v1.Worker, vms []v1.VM) {
 	for _, worker := range workers {
+		workersLastSeen.With(map[string]string{"worker_name": worker.Name}).Set(float64(worker.LastSeen.Unix()))
 		if worker.Offline(scheduler.workerOfflineTimeout) {
+			workersStatus.With(map[string]string{"worker_name": worker.Name, "status": "online"}).Set(0)
```

Comment:
Do you think we need a separate `"status": "offline"` increment here and `.Set(0)` in `else statement?

With only online status it seems you'll need to change the alert each time you add a node. And there are two options:

1. Add an offline status
2. Don't report this metric and then have a alert on an absent metrics.

I think #1 makes sense. What do you think?

Reply:
Sure, makes sense!